src/ts/IconWrap/index.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { FontIcon } from 'material-ui';
+
+export default class IconWrap extends React.Component<any, any> {
+  state = { isMounted: false };
+
+  componentDidMount() {
+    this.setState({ isMounted: true });
+  }
+
+  render() {
+    if (this.state.isMounted) return <FontIcon className="material-icons" style={this.props.style}>{this.props.icon}</FontIcon>;
+    return <div aria-label={this.props.aria && ''} className="hide">{this.props.aria && ''}</div>;
```

Comment:
good point, I meant `aria || ''`, because in other way it's render as string `undefined`